refactor: replace the docs-code with inline code and fix the indent

aparzi opened the pull request:
Replace the docs-code with inline code and fix the indent

fixes #65148

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/contributing-docs/commit-message-guidelines.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.dev application / infrastructure changes
- [ ] Other... Please describe:

Issue Number: #65148


## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

JeanMeche commented on adev/src/content/guide/components/queries.md:
```diff
@@ -1,6 +1,7 @@
 
-TIP: This guide assumes you've already read the [Essentials Guide](essentials). Read that first if you're new to Angular.
+TIP: This guide assumes you've already read the [Essentials Guide](essentials). Read that first if you're new to
+Angular.
```
This change is unexpected / not necessary.

JeanMeche commented on adev/src/content/guide/components/queries.md:
```diff
@@ -13,9 +14,11 @@ There are two categories of query: **view queries** and **content queries.**
 
-View queries retrieve results from the elements in the component's _view_ — the elements defined in the component's own template. You can query for a single result with the `viewChild` function.
+View queries retrieve results from the elements in the component's _view_ — the elements defined in the component's own
+template. You can query for a single result with the `viewChild` function.
```
This change is unexpected / not necessary.

JeanMeche commented on adev/src/content/guide/components/queries.md:
```diff
@@ -79,34 +85,40 @@ export class CustomToggle {
 
-If the query does not find a result, its value is `undefined`. This may occur if the target element is absent or hidden by `@if`. Angular keeps the result of `contentChild` up to date as your application state changes.
+If the query does not find a result, its value is `undefined`. This may occur if the target element is absent or hidden
+by `@if`. Angular keeps the result of `contentChild` up to date as your application state changes.
```
This change is unexpected / not necessary.

JeanMeche commented on adev/src/content/guide/components/queries.md:
```diff
@@ -338,36 +372,41 @@ export class CustomToggle {
-export class UserProfile { }
-</docs-code>
+export class UserProfile {
+}
```
We can probably have that on a single line

JeanMeche commented on adev/src/content/guide/components/queries.md:
```diff
@@ -50,26 +52,28 @@ export class CustomCardAction {
 
-Content queries retrieve results from the elements in the component's _content_— the elements nested inside the component in the template where it's used. You can query for a single result with the `contentChild` function.
+Content queries retrieve results from the elements in the component's _content_— the elements nested inside the
+component in the template where it's used. You can query for a single result with the `contentChild` function.
```
This should be reformated. (there are multiple cases in this PR)

JeanMeche commented on adev/src/content/guide/components/queries.md:
```diff
@@ -240,18 +257,21 @@ export class CustomExpando {
-export class UserProfile { }
-</docs-code>
+export class UserProfile {
+}
```
nit
```suggestion
export class UserProfile {}
```

JeanMeche commented on adev/src/content/guide/components/queries.md:
````diff
@@ -367,7 +374,7 @@ Angular keeps the result of `@ContentChild` up to date as your application state
 You can also query for multiple results with the `@ContentChildren` decorator.
 
-<docs-code language="angular-ts" highlight="[14, 16, 17, 18, 19, 20]">
+```typescript {highlight: [14, 16, 17, 18, 19, 20]}
````
Same here, we need to update the lines

aparzi replied:
I updated the lines

aparzi committed:
refactor: convert docs-code blocks to inline TypeScript documentation

This PR converts the custom <docs-code> blocks to standard Markdown code blocks with TypeScript syntax highlighting.

fixes #65148

JeanMeche commented on adev/src/content/guide/components/queries.md:
```diff
@@ -79,34 +80,34 @@ export class CustomToggle {
 }
 
 @Component({
 /_ ... _/
```
```suggestion
/* ... */
```

aparzi committed:
fixup! refactor: convert docs-code blocks to inline TypeScript documentation
